mne/decoding/time_gen.py

```diff
@@ -139,11 +143,11 @@ class GeneralizationAcrossTime(object):
 
     .. versionadded:: 0.9.0
     """  # noqa
-    def __init__(self, cv=5, clf=None, train_times=None,
+    def __init__(self, cv=5, clf=None, train_times=None, predict_shape=1,
                  predict_mode='cross-validation', n_jobs=1):
 
         from sklearn.preprocessing import StandardScaler
-        from sklearn.svm import SVC
+        from sklearn.linear_model import SGDClassifier
```

Comment:
Logistic regression is more explicit to people outside the ML community (how cheap is this argument...) Let's keep it like that; it works, it's good.